packages/scipy/meta.yaml

```diff
@@ -50,6 +53,9 @@ build:
   # pyodide-build/pyodide_build/_f2c_fixes.py.
   script: |
     set -x
+    git clone https://github.com/hoodmane/f2c.git --depth 1
+    (cd f2c/src && touch gram.c && cp makefile.u makefile && make)
+    export F2C_PATH=$(pwd)/f2c/src/f2c
```

Comment:
We can remove `f2c` from `requirements.executable`, if we are going to build it here.